pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -594,6 +606,7 @@ func (gc *GarbageCollector) apiResource(apiVersion, kind string, namespaced bool
 func (gc *GarbageCollector) deleteObject(item objectReference) error {
 	fqKind := unversioned.FromAPIVersionAndKind(item.APIVersion, item.Kind)
 	client, err := gc.clientPool.ClientForGroupVersion(fqKind.GroupVersion())
+	gc.registeredRateLimiter.register(fqKind.GroupVersion(), client, "garbage_collector_operation")
```

Comment:
Here and the next three changes register the restclients used to send out delete/get/update/patch operations.